### PR TITLE
 #339 - Option: Skip initial load of DataBehavior 

### PR DESCRIPTION
**Pull request by mandragorn**
* skipInitialLoad (defaults to false) when set to true will skip the
  initial load of the data behavior when the view completes
  initialization.

needed to include #307 for now, but will rebase once it is merged.

### CODE REVIEW

**Comment by mandragorn** [comment]
@kentmw - this should be the last one for this week

File: test/karma/dataBehaviorSpec.js [hunk 1]
```diff
@@ -131,6 +131,54 @@ describe('A Torso Data Behavior', function() {
...
+      var ViewWithDataBehaviorSkipInitialLoadTrue = TorsoView.extend({
+        behaviors: {
+          dataBehavior: defaultBehaviorConfigurationSkipInitialLoadTrue
+        }
+      });
+      var viewWithDataBehaviorSkipInitialLoadTrue = new ViewWithDataBehaviorSkipInitialLoadTrue();
+      expect(viewWithDataBehaviorSkipInitialLoadTrue.getBehavior('dataBehavior').skipInitialLoad).toBe(true);
+    });
+
+    it('defaults returnSingleResult to false', function() {
```
Comment (fyliu24) [inline test/karma/dataBehaviorSpec.js]: Is this a duplicate? Looks the same as the one below?

Comment (mandragorn) [reply test/karma/dataBehaviorSpec.js]: fixed

**Comment by mandragorn** [comment]
rebase

**Pull request merged**